browser/about_flags.cc

```diff
@@ -478,6 +482,16 @@
                    FEATURE_VALUE_TYPE(brave_player::features::kBravePlayer), \
                }))
 
+#define BRAVE_EXTENSIONS_MANIFEST_V2                                        \
+  IF_BUILDFLAG(ENABLE_EXTENSIONS,                                           \
+               EXPAND_FEATURE_ENTRIES({                                     \
+                   "brave-extensions-manifest-v2",                          \
+                   "Brave Extensions manifest V2",                          \
+                   "Enables Brave support for some manifest V2 extensions", \
```

Comment:
Right now this sounds like Mv2 will not function for any extensions without the flag enabled. Maybe we can explicitly mention that this is for adding extension install options to the settings?